README.md

```diff
@@ -4,7 +4,15 @@
 
 1. Submit a new [access request issue](https://github.com/rust-vmm/community/issues/new?assignees=andreeaflorescu%2C+jbryce%2C+sameo&labels=&template=access-request-to-rust-vmm-organization.md&title=Access+Request+to+rust-vmm).
 
-## Adding a new repository
+## How to contribute?
+
+### Adding a new crate to rust-vmm
+
+Each rust-vmm crate lives in its own repo under the github.com/rust-vmm/ organization.
+If you want to contribute to rust-vmm by adding a new Rust crate to the project, you will have to go through the following steps:
+
+1. Submit a [crate addition request](https://github.com/rust-vmm/community/issues/new?assignees=&labels=&template=new-crate-request.md&title=Crate+Addition).
```

Comment:
Should we use the review request template instead? https://github.com/rust-vmm/community/issues/new?assignees=&labels=&template=repository-review-request.md&title=Repository%2FCrate+Review+Request

Reply:
Nvm, I just saw you modified the template as well.